core/src/main/scala/kafka/log/Log.scala

```diff
@@ -170,9 +172,14 @@ class Log(val dir: File,
     // now do a second pass and load all the .log and .index files
     for(file <- dir.listFiles if file.isFile) {
       val filename = file.getName
-      if(filename.endsWith(IndexFileSuffix)) {
+      if(filename.endsWith(IndexFileSuffix) || filename.endsWith(TimeIndexFileSuffix)) {
```

Comment:
The comment on line 172 is now outdated. Perhaps we can just say all index files.